app/models/facility.rb

```diff
@@ -234,4 +231,39 @@ def localized_facility_size
     return unless facility_size
     I18n.t("activerecord.facility.facility_size.#{facility_size}", default: facility_size.capitalize)
   end
+
+  def locale
+    LOCALE_MAP.dig(country.downcase, state.downcase) || LOCALE_MAP.dig(country.downcase, "default") || LOCALE_MAP["default"]
+  end
+
+  LOCALE_MAP = {
```

Comment:
We can extract this into a service object in the future if we ever need it for anything other than facility.